communications/github-projects.md

```diff
@@ -0,0 +1,59 @@
+## On Github Projects
+
+Since the [Covid Dash development](https://github.com/EQWorks/flaat-dash/projects), we have been using github projects to keep track of progress, issues, and code-specific discussions due to the fast paced nature of the project. 
+
+Github Projects tool can be used in any other repository. Repos can have multiple projects. Projects can be specific to one extensive task or the whole repo in general - ideally both. Projects that have issues intertwined with different repositories (for instance front-end repo and back end repo) can take advantage of the cross-repo issue referencing. 
+
+<img width="800" alt="column" src=https://user-images.githubusercontent.com/53827690/83157518-b1cf8e00-a0d1-11ea-9b2c-7a4019ae5066.png>
+
+Projects are a to-do list: easy to keep track of progress as it shows the percentage between open issues / in progress issues / completed issues.
+
+<img width="800" alt="column" src=https://user-images.githubusercontent.com/53827690/83158139-6964a000-a0d2-11ea-9e37-b3f6edd0b7f4.png>
+
+The project is fully customizable and the user can add or delete columns relevant to the project
+<p align="center">
+    <img width="300" alt="column" src=https://user-images.githubusercontent.com/53827690/83159389-ec3a2a80-a0d3-11ea-89fa-7e112f77747d.png>
+    <img width="310" alt="column" src=https://user-images.githubusercontent.com/53827690/83159809-68cd0900-a0d4-11ea-8a00-d6bb168708bb.png>
+</p>
```

Comment:
May want to add about automation since some columns can be automated to move cards to another column once their status changes. i.e. once PR is reviewed, it moves into 'In Review' column automatically.

Reply:
does it? I've never experienced that, only that once the PR is merged and its description has the keyword close the issue, it will move to `done` but I've never seen it moving to `in review` automatically.